test/Table_Tests/src/Table_Time_Spec.enso

```diff
@@ -0,0 +1,52 @@
+from Standard.Base import all
+import Standard.Base.Data.Text.Line_Ending_Style
+import Standard.Base.Data.Time
+
+import Standard.Table
+import Standard.Table.Data.Column
+import Standard.Table.IO.File_Format
+from Standard.Table.Data.Data_Formatter as Data_Formatter_Module import Data_Formatter
+
+import Standard.Test
+
+from project.Util import all
+
+spec =
```

Comment:
Should we have one more suite like that for `Date`?

Reply:
There is one already

Reply:
Ah sorry, I just looked at the changes and since it was unchanged it didn't show up